src/desktop/apps/notifications/templates/mobile/artist.jade

```diff
@@ -9,4 +9,4 @@
       | works added
   .notifications-published-artworks
   .notifications-see-more
-    a.chevron-arrow-button( href=artist.href() + '?sort=-published_at' ) See More
```

Comment:
I'm not sure why this was named `chevron-arrow-button`, there was no chevron or arrow or button.

Reply:
Lol, like the Holy Roman Empire...